octodns_azure/__init__.py

```diff
@@ -539,6 +545,7 @@ def _dns_client(self):
                     logger=logger,
                 ),
                 subscription_id=self._dns_client_subscription_id,
+                retry_policy=self._dns_client_retry_policy
```

Comment:
Should we also set equivalent of this for `self.__tm_client`?

Reply:
Let me look into it :). Also thank you both for reviewing already!

Reply:
It looks like it doesn't implement the retry_policy? 🤔 
https://docs.microsoft.com/en-us/python/api/azure-mgmt-trafficmanager/azure.mgmt.trafficmanager.trafficmanagermanagementclient?view=azure-python

Reply:
Looking at the code, the `TrafficManagerClient` does take `kwargs` (annoying that its not documented) which gets funneled to a chain of various other objects internally that are indeed respecting `retry_policy`. Even if not, by virtue of `kwargs`, an unknown key will just get ignored, so passing the same `retry_policy` to `__tm_client` sounds harmless.

I'd prefer to keep stuff uniform across the various `*Client` objects as much as possible.

Reply:
Sounds good to me. Will add it 👍

Reply:
Added via 7b413ac